docs/cne-install.md

```diff
@@ -0,0 +1,177 @@
+  procedure is run.
+
+### Notes on the Procedures
+
+- Ellipses (`...`) in shell output indicate omitted lines.
+- In the examples below, replace `x.y.z` with the version of the SAT product stream
+  being upgraded.
+- 'manager' and 'master' are used interchangeably in the steps below.
+
+### Pre-Upgrade Procedure
+
+1.  Start a typescript and set the shell prompt.
+
+    The typescript will record the commands and the output from this upgrade.
+    The prompt is set to include the date and time.
+
+    ```screen
+    ncn-m001# script -af product-sat.$(date +%Y-%m-%d).txt
+    ncn-m001# export PS1='\u@\H \D{%Y-%m-%d} \t \w # '
+    ```
+
+### Upgrade Procedure
+
+1.  Copy the release distribution gzipped tar file to `ncn-m001`.
+
+    Whether you are upgrading SAT alone or with other supported products, copy the file in the same media directory as all other supported products.
+
+1.  Run the CNE installer.
+
+    - If you are upgrading SAT along with other supported products, run the following command.
+
+        ```screen
+        ncn-m001# cne-install -m MEDIA_DIR install -B WORKING_BRANCH -bpc BOOTPREP_CONFIG_CN -bpn BOOTPREP_CONFIG_NCN
+        ```
+
+        The `cne-install` command will use the provided `BOOTPREP_CONFIG_CN` and `BOOTPREP_CONFIG_NCN` files for the run.
+
+    - If you are upgrading SAT alone, run the following command.
+
+        ```screen
+        ncn-m001# cne-install -m MEDIA_DIR install -B WORKING_BRANCH
+        ```
+
+1.  **Optional:** Stop the typescript.
+
+    **NOTE**: This step can be skipped if you wish to use the same typescript
+    for the remainder of the SAT upgrade. See [Next Steps](#next-steps).
+
+    ```screen
+    ncn-m001# exit
+    ```
+
+SAT version `x.y.z` is now upgraded, meaning the SAT `x.y.z` release
+has been loaded into the system software repository.
+
+- SAT configuration content for this release has been uploaded to VCS.
+- SAT content for this release has been uploaded to the CSM product catalog.
+- SAT content for this release has been uploaded to Nexus repositories.
+- The `sat` command is available.
+
+### Next Steps
+
+At this point, the release distribution files can be removed from the system as
+described in [Post-Upgrade Cleanup Procedure](#post-upgrade-cleanup-procedure).
+
+If other HPE Cray EX software products are being upgraded in conjunction
+with SAT, refer to the [*HPE Cray EX System Software Getting Started Guide (S-8000)*](https://www.hpe.com/support/ex-S-8000)
+to determine which step to execute next.
+
+If no other HPE Cray EX software products are being upgraded at this time,
+execute the **SAT Post-Upgrade** procedures:
+
+- [Remove obsolete configuration file sections](#remove-obsolete-configuration-file-sections)
+- [SAT Logging](#sat-logging)
+
+## Post-Upgrade Cleanup Procedure
+
+1.  **Optional:** Remove the SAT release distribution tar file and extracted directory.
+
+    ```screen
+    ncn-m001# rm sat-x.y.z.tar.gz
+    ncn-m001# rm -rf sat-x.y.z/
+    ```
```

Comment:
Same guidelines as above:

~~~suggestion
1. **Optional:** Remove the SAT release distribution tar file and extracted directory.

   ```screen
   ncn-m001# rm sat-x.y.z.tar.gz
   ncn-m001# rm -rf sat-x.y.z/
   ```
~~~